packages/launchpad/src/components/topnav/TopNav.vue

```diff
@@ -0,0 +1,294 @@
+<template>
+  <TopNavList v-if="versionList">
+    <template #heading="{ open }">
+      <i-cy-box_x16
+        class="group-hocus:icon-dark-indigo-500 group-hocus:icon-light-indigo-50 h-16px w-16px"
+        :class="open ? 'icon-dark-indigo-500 icon-light-indigo-50' : 'icon-dark-gray-500 icon-light-gray-100'"
+      />
+      <span>v{{ versionList[0].version }}</span>
+    </template>
+    <TopNavListItem
+      v-for="(item, index) in versionList"
+      :key="item.version"
+      :selectable="!!index"
+      class="px-4 py-2 min-w-240px"
+      :class="index ? '' : 'bg-jade-50'"
+    >
+      <div class="whitespace-nowrap">
+        <a
+          :href="`${releasesUrl}/tag/v${item.version}`"
+          :class="index ? '' : 'text-jade-600'"
+          class="font-semibold"
+          target="_blank"
+        >{{ item.version }}</a>
+        <br>
+        <span class="text-gray-600 text-14px">Released {{ item.released }}</span>
+      </div>
+      <template
+        v-if="!index"
+        #suffix
+      >
+        <i-cy-circle-check_x24
+          class="icon-dark-jade-100 icon-light-jade-500 w-24px h-24px"
+        />
+      </template>
+    </TopNavListItem>
+    <TopNavListItem class="p-4 text-center bg-gray-50">
+      <a
+        :href="releasesUrl"
+        target="_blank"
+        class="block w-full py-2 border-gray-100 text-14px whitespace-nowrap border-rounded border-1 hover:no-underline hover:border-gray-200"
+      >See all releases</a>
+    </TopNavListItem>
+  </TopNavList>
+
+  <TopNavList v-if="props.gql?.selectedBrowser && props.showBrowsers">
+    <template #heading="{ open }">
+      <img
+        class="w-16px filter group-hocus:grayscale-0"
+        :class="open ? 'grayscale-0' : 'grayscale'"
+        :src="allBrowsersIcons[props.gql?.selectedBrowser?.displayName || '']"
+      >
+      <span>{{ props.gql.selectedBrowser?.displayName }} v{{ props.gql.selectedBrowser?.majorVersion }}</span>
+    </template>
+    <TopNavListItem
+      v-for="browser in props.gql.browsers"
+      :key="browser.id"
+      class="px-4 py-3 min-w-240px"
+      :class="browser.isSelected ? 'bg-jade-50' : ''"
+      :selectable="!browser.isSelected"
+    >
+      <template #prefix>
+        <!-- setting both width and min-width on these icons looks odd,
+        but makes all possible browser icons happy about what size to be -->
+        <img
+          class="mr-4 min-w-26px w-26px"
+          :src="allBrowsersIcons[browser.displayName]"
+        >
+      </template>
+      <div>
+        <div
+          :class="browser.isSelected ? 'text-jade-600' : 'text-indigo-600'"
+        >
+          {{ browser.displayName }}
+        </div>
+        <div
+          class="mr-6 font-normal text-gray-500 whitespace-nowrap text-14px"
+        >
+          Version {{ browser.version }}
```

Comment:
FYI I would've added i18n here. It's probably fine without it, but see Experiments.vue for usage...

```vue
<i18n-t keypath="settingsPage.experiments.description">
  <a
    href="https://docs.cypress.io"
    target="_blank"
  >{{ t('links.learnMore') }}</a>
</i18n-t>
```

Generates:
```markdown
If you'd like to try out new features that we're working on,
you can enable beta features for your project by turning on
the experimental features you'd like to try. <a href="...">Learn More.</a>
```

Reply:
I'm all for doing i18n as we go so it's not a massive chore moving every string in there at some point.